### DISCUSSION TITLE
Someone mentioned UTM page used to have Win 8 and 8.1 templates - Gone - Can you share again please?

### DISCUSSION THREAD

**Discussion by xbliss** [utmapp/UTM#4103] [UTM Help (macOS)]
Someone mentioned UTM page used to have Win 8 and 8.1 templates - Gone - Can you share again please? 

https://mac.getutm.app/gallery/

**Comment by osy** [comment]
I don’t think there’s ever been one but it should be the same steps as windows 7.
